src/pages/BuildPlanner.vue

```diff
@@ -7,25 +7,30 @@
       <q-step name="buildEnvironment"
               title="Build environment"
               style="height: auto;"
-              :error="buildPlan.platforms.length < 1" error-color="negative">
+              :done="currentStep != 'buildEnvironment'"
+              :error="checkError()" error-color="negative">
 
         <q-checkbox left-label class="text-grey-8 text-body1 q-pb-sm"
                     v-model="buildPlan.is_secure_boot" label="Secure Boot"/>
 
         <q-select v-model="product"
                   :options="buildProducts"
-                  label="Product:"
+                  label="Product*:"
```

Comment:
Can't we indicate that this field is required differently? Visually, the * looks a bit weird to me.

![image](https://user-images.githubusercontent.com/821790/191805761-4f247986-5ba7-42af-81eb-8a746c441305.png)

As an idea, can't we add the hint "This field is required" (or something similar) in the same way we do with linked builds? Then, make the hint disappear when the field is provided.

Reply:
Yea, thank you 🚀